src/components/Slider/components/SliderCard/components/SliderCardFooter/SliderCardFooter.tsx

```diff
@@ -3,12 +3,18 @@ import { Link } from 'react-router-dom';
 import { Button, CardActions } from '@mui/material';
 import './SliderCardFooter.scss';
 
-export const SliderCardFooter = () => (
+interface SliderCardFooterProps {
+  id: string;
+}
```

Comment:
move interface to common directory of types

Reply:
why i thought props interface can be into component folder

Reply:
I think all Interfaces should be in types(also for Props)
It is more convenient and  you can extend them or reuse

Reply:
Props interfaces can be left in the same file